src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.VisualBasic.xaml

```diff
@@ -450,11 +450,12 @@
                   DisplayName="Custom constants"
                   HelpUrl="https://go.microsoft.com/fwlink/?linkid=2196264"
                   Category="Advanced"
-                  Description="Specifies custom conditional compilation constants for your application.">
+                  Description="Specifies custom conditional compilation constants for your application. Note that values are optional, for the optional behavior leave the value blank. If the value is a string, please add quotation marks around it.">
```

Comment:
So a constant with no value would be written as "Value=" or "Value"?

Reply:
yup there are two options: "value" or "value=thing"

Reply:
Should update the message then. It explains as if it should be "left=" as if value is blank